src/main/java/com/jamierf/dropwizard/logging/gelf/GelfAppenderFactory.java

```diff
@@ -79,6 +121,12 @@ public void setServer(HostAndPort server) {
         }
 
         appender.getConfig().getDefaultFields().put(GelfTargetConfig.FIELD_FACILITY, applicationName);
+        appender.getConfig().getDefaultFields().putAll(defaultFields);
+        if(!additionalFields.isEmpty())
+        {
+            appender.getConfig().getAdditionalFields().clear();
```

Comment:
It clears the defaults for additional fields, in the same manner the gelf4j library does. If additional fields are specified by the user those supercede the defaults.